packages/esm-patient-labs-app/src/lab-orders/add-lab-order/lab-order-form.scss

```diff
@@ -80,3 +80,8 @@
     margin-bottom: calc(var(--bottom-nav-height) + 1rem);
   }
 }
+
+
+.topOfFormSlot {
+  margin: spacing.$spacing-03 spacing.$spacing-03;
```

Comment:
This margin will be applied regardless of whether the slot has any content. Can we obviate that?

Reply:
Resolved this concern.